sql-jdbc/build.gradle

```diff
@@ -51,11 +51,11 @@ dependencies {
 
     testImplementation('org.junit.jupiter:junit-jupiter-api:5.3.1')
     testImplementation('org.junit.jupiter:junit-jupiter-params:5.3.1')
-    testImplementation('com.github.tomakehurst:wiremock:2.27.2')
+    testImplementation "com.github.tomakehurst:wiremock-jre8-standalone:2.34.0"
     testImplementation('org.mockito:mockito-core:2.23.0')
     testImplementation('org.junit.jupiter:junit-jupiter-engine:5.3.1')
     testImplementation('org.junit-pioneer:junit-pioneer:0.3.0')
-    testImplementation('org.eclipse.jetty:jetty-server:9.2.24.v20180105')
+    testImplementation('org.eclipse.jetty:jetty-server:11.0.11')
```

Comment:
How come jetty-server is upgraded to `11.x` instead of `9.4.48.v20220622`. That's the version that wiremock 2.34.0 depends on.

Are the changes in TLSServer.java necessary because of Jetty 9 vs Jetty 11 differences?

Reply:
Thanks for the tip!